cmd/dump.go

```diff
@@ -231,11 +238,39 @@ func runDump(ctx *cli.Context) error {
 			fatal("Failed to include repositories: %v", err)
 		}
 
-		if _, err := os.Stat(setting.LFS.ContentPath); !os.IsNotExist(err) {
-			log.Info("Dumping lfs... %s", setting.LFS.ContentPath)
-			if err := addRecursive(w, "lfs", setting.LFS.ContentPath, verbose); err != nil {
-				fatal("Failed to include lfs: %v", err)
+		switch setting.LFS.Storage.Type {
+		case setting.LocalStorageType:
+			if _, err := os.Stat(setting.LFS.Path); !os.IsNotExist(err) {
+				log.Info("Dumping lfs... %s", setting.LFS.Path)
+				if err := addRecursive(w, "lfs", setting.LFS.Path, verbose); err != nil {
```

Comment:
addRecursive doesn't appear to be much better than just using IterateLFS

Reply:
See https://github.com/lunny/gitea/pull/7/commits/fe22860c7dcb7e59ec9d98cf6895c43e08fce7c1